router/routes/pew.js

```diff
@@ -23,7 +26,11 @@ router.post('/', function(request, response) {
   if (!input || input.length === 0) {
     message.push("now");
   } else {
-    message.push("in", input, "minutes");
+    parsed = input.split(' ');
+    minutes = parsed[0];
+    game = parsed[1];
+    message.push(game, "in", minutes, "minutes");
+    _.compact(message);
```

Comment:
This won't assign the result of `compact` to message. It returns a new array. If you want message to be the compacted array, you have to do `message = _.compact(message);`